### ISSUE TITLE
Warn users if they are using outdated Python versions.

### ISSUE THREAD

**Issue by anders314159** [TheRacetrack/plugin-python-job-type#48]
Is there anything warning users if they are using old versions of Python in their job-types? Old defined as end-of-life by https://devguide.python.org/versions/

Do we want to implement that?

**Comment by anders314159** [comment]
@JosefAssad

**Comment by iszulcdeepsense** [comment]
I think we can do that, but mostly because it's easy to do. 
It feels a bit odd with regard to responsibility of this job type. If we're going to do this, what's next? Do we want to find vulnerabilities in third-party libs? I guess not, but where's the limit?

**Comment by anders314159** [comment]
I agree - I'll bring the issue up at the huddle, and hear if it is actually a problem or just busy-work.

**Comment by anders314159** [comment]
ML Lab said it would be very nice to have. We'll limit it to just the python version for now, not checking requirements.txt or anything else for outdated packages.
